src/mixins/filing-template-mixin.ts

```diff
@@ -405,6 +408,55 @@ export default class FilingTemplateMixin extends DateMixin {
     return filing
   }
 
+  /**
+   * Builds a restoration filing from store data. Used when saving a filing.
+   * @returns the filing body to save
+   */
+  buildRestorationFiling (): any {
+    // Build the main filing.
+    const filing: RestorationFilingIF = {
+      header: {
+        name: FilingTypes.RESTORATION,
+        certifiedBy: this.getCertifyState.certifiedBy,
+        date: this.getCurrentDate,
+        filingId: this.getFilingId,
+        folioNumber: this.getFolioNumber, // default FN; may be overwritten by staff BCOL FN
+        isFutureEffective: false
+      },
+      business: {
+        legalType: this.getEntityType,
+        identifier: this.getBusinessId,
+        legalName: this.getBusinessLegalName,
+        foundingDate: this.getBusinessFoundingDate
+      },
+      // *** TODO: update
```

Comment:
This is for future tickets to update (as they add data, etc).

Ditto for parseRestorationDraft() below.